src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
```

Comment:
Consider using only top or bottom margin to avoid potential margin collapse. The checklist suggests being consistent with your margins.

Reply:
I have couple comments here:

1) There is nothing stated in the task to set horizontal margins, I had to ask mentor to understand why tests were failing and he suggested putting "margin: 0 10px", but at 10px tests still were failing. I had to find the correct margin to comply the tests. I believe this point should be added to task description or tests should be updated

2) Why the bot says it's incorrect margins? I was suggested by mentor to set margins this way?

Reply:
@etojeDenys please check these questions